site/reference/fn/doc/README.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Doc"
+linkTitle: "doc"
+type: docs
+description: >
+    Display the documentation for a function
+---
+
+<!--mdtogo:Short
+    Display the documentation for a function
+-->
+
+### Examples
+
+<!--mdtogo:Examples-->
+
+```shell
+# diplay the documentation for image gcr.io/kpt-fn/set-namespace:v0.1.1
+kpt fn doc --image gcr.io/kpt-fn/set-namespace:v0.1.1
+```
+
+<!--mdtogo-->
+
+### Synopsis
+
+<!--mdtogo:Long-->
+
+`kpt fn doc` invokes `docker run` to extract the help text. It requires the
```

Comment:
docker is a registered trademark and an implementation of container orchestration. Whether kpt invokes docker or not is an implementation detail. What you want to convey ehre:

X invokes the function container with `--help` flag ...

Reply:
Updated.